source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -349,6 +407,32 @@ TEST_F(NiDCPowerDriverApiTest, NiDCPowerSetViInt64Attribute_SendRequest_GetViInt
   EXPECT_EQ(expected_value, get_attribute_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, VoltageLevelConfiguredAndExportedToBuffer_ResetAndImportConfigurationFromBuffer_ConfigurationIsImportedSuccessfully)
+{
+  const char* channel_name = "0";
+  ViReal64 expected_voltage_level = 3.0;
+  configure_output_function(channel_name, dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE);
+  configure_voltage_level(channel_name, expected_voltage_level);
+  auto export_buffer_response = export_attribute_configuration_buffer();
+
+  reset_with_channels(channel_name);
+  ::grpc::ClientContext context;
+  dcpower::ImportAttributeConfigurationBufferRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  for (int i = 0; i < export_buffer_response.configuration_size(); i++)
```

Comment:
Use iterators to do the copy:
```cpp
auto exported_configuration = export_buffer_response.configuration();
import_request.add(exported_configuration.begin(), exported_configuration.end());
```
Then consider factoring out the copy step into a helper that takes in the export response & import request as parameters.